packages/request-client.js/test/index.test.ts

```diff
@@ -1917,7 +1989,7 @@ describe('index', () => {
         signatureProvider: TestData.fakeSignatureProvider,
         useMockStorage: true,
       });
-      const salt = 'ea3bc7caf64110ca';
+      const salt = 'ea3bc7caf64110cb';
```

Comment:
this is actually not related, but imo this test makes no sense: it's about creating a request WITH a salt. But, since this salt has already been used, the balance is not 0... I changed the salt so that this test focuses on what it is supposed to